HHS/Head-Start-TTADP
# [Prod] Add UI for national centers, fix TR report permissions, and fix bug with removing recipients after adding files

## Description of change

- Many improvements to TR form based on feedback, including permissions overhaul
- Fix bug when removing recipients from report with files
- Add a DB table + an admin UI for editing National centers

## How to test

- Fully create an activity report with many recipients, adding one goal and objective with files. Confirm everything is saved properly. You can remove a recipient without error.
- Confirm the session report still displays the proper "objective trainers." View the national center admin UI, confirm you can edit and save items there.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1862
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1856
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1808

## Checklists

### Every PR


- [n/a] API Documentation updated
- [n/a] Boundary diagram updated
- [n/a] Logical Data Model updated
- [n/a] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions
- [n/a] UI review complete

### Production Deploy


### After merge/deploy
